### ISSUE TITLE
Descriptions are not generated from JavaDocs

### ISSUE THREAD

**Issue by solomkinmv** [ScaCap/spring-auto-restdocs#86]
I've configured the project and generated docs. Everything looked fine. But when I added javadocs and generated docs nothing changed.
What is the reason for such problem?
Here is by [build.gradle](https://github.com/solomkinmv/glossary/blob/auto-docs/web/build.gradle) and [tests](https://github.com/solomkinmv/glossary/blob/auto-docs/web/src/test/java/io/github/solomkinmv/glossary/web/controller/WordControllerTest.java).

**Comment by fbenz** [comment]
Issue is that you are only extracting Javadoc in the `web` module, but the Javadoc you want is in the `persistence` module. Javadoc extraction has to be set up for every module that contains classes that are used in the resources. Afterwards you have two options on using the Javadoc from the `persistence` module in the build of the `web` module:
1. Add the `generated-javadoc-json` folder from the `persistence` module to the `org.springframework.restdocs.javadocJsonDir` system property (comma-separated list of folders).
2. Copy over the files from the `generated-javadoc-json` folder from the `persistence` module to the `generated-javadoc-json` folder in the `web` module.

**Comment by solomkinmv** [comment]
I've added some sample javadocs to the method in the controller ([link](https://github.com/solomkinmv/glossary/blob/auto-docs/web/src/main/java/io/github/solomkinmv/glossary/web/controller/WordController.java#L50)), but there is no description for this parameter.
What am I doing wrong?
![image](https://cloud.githubusercontent.com/assets/9872823/23878057/e5d21198-084d-11e7-9766-c00aabaa4495.png)

**Comment by jmisur** [comment]
When I checked out your branch and generated docs, I indeed see parameter description. Make sure `glossary/web/build/generated-javadoc-json/io/github/solomkinmv/glossary/web/controller/WordController.json` has description for query. For me it has. Another problem would be the response, which is a hateoas wrapper around object, that we don't support out of the box yet.
![screen shot 2017-03-15 at 18 09 30](https://cloud.githubusercontent.com/assets/321761/23961320/dd2ed612-09aa-11e7-8374-5dd41ad3e231.png)
